tests/all/unit/api/helpers/test_utilities.py

```diff
@@ -17,6 +17,20 @@ def test_get_filename_from_cd(self):
         self.assertEqual(expected_response_first, response_first)
         self.assertEqual(expected_response_none, response_none)
 
+    def test_string_empty(self):
+        """Method to test whether an empty string is correctly identified."""
+
+        self.assertTrue(string_empty(''))
+        self.assertTrue(string_empty(' '))
+        self.assertTrue(string_empty('  '))
+        self.assertTrue(string_empty('\n'))
```

Comment:
Add tabs as well

Reply:
The third test case is a tab but will make it into multiple tabs

Reply:
Added